js/sitnlayers.js

```diff
@@ -68,26 +81,30 @@
         src: _markerImage,
       });
     };
-    // TODO: get out source from here when base layer selection will be implemented
+
     sitnLayers.setSource = function (layerName) {
-      fetch(_WMTSurl).then((response) => response.text()).then((getCap) => {
-        const _parser = new ol.format.WMTSCapabilities();
-        const _result = _parser.read(getCap);
-        sitnLayers.WMTSOptions = ol.source.WMTS.optionsFromCapabilities(
-          _result, {
-          layer: layerName,
-          matrixSet: _crs,
-        },
-        );
-        const source = new ol.source.WMTS(sitnLayers.WMTSOptions);
-        sitnLayers.sitnCurrentBaseLayer.setSource(source);
-      });
+      const source = new ol.source.WMTS({
+        layer: layerName,
+        attributions: 'Informations dépourvues de foi publique, '
+          + '<a target="new" href="https://sitn.ne.ch/web/conditions_utilisation/contrat_SITN_MO.htm">'
+          + '© SITN</a> | '
+          + '<a target="new" href="https://sitn.ne.ch/web/conditions_utilisation/contratdv5741.htm">'
+          + 'swisstopo DV 571.4</a> | '
+          + '<a href="http://www.openstreetmap.org/copyright" target="_blank">OpenStreetMap</a> | '
+          + '<a href="https://mapicons.mapsmarker.com/about/license/" target="_blank">Maps Icons Collection</a>',
+        projection: _projection,
+        url: 'https://sitn.ne.ch/mapproxy95/wmts/1.0.0/{layer}/default/EPSG2056/{TileMatrix}/{TileRow}/{TileCol}.png',
```

Comment:
Is that working with orthophotos? They are served in the jpeg format...

Reply:
Thanks, fixed it!